check-before-deployment.md

```diff
@@ -229,6 +229,103 @@ sudo systemctl start ntpd.service && \
 sudo systemctl enable ntpd.service
 ```
 
+## 检测及关闭透明大页
+
+对于数据库应用，不推荐使用 THP（transparent hugepage，透明大页），因为数据库往往具有稀疏而不是连续的内存访问模式，且当高阶内存碎片化比较严重时，分配 THP 页面会出现较大的延迟。若开启针对 THP 的直接内存规整功能，也会出现系统 CPU 使用率激增的现象，因此建议关闭 THP。
+
+采用如下步骤检查是否已经关闭透明大页：
+
+1. 执行以下命令，如果输出 `[always] madvise never` 表示透明大页处于启用状态：
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    cat /sys/kernel/mm/transparent_hugepage/enabled
+    ```
+
+    ```
+    [always] madvise never
+    ```
+
+2. 执行 `grubby` 命令查看默认内核版本：
+
+    > **注意：**
+    >
+    > 需安装 `grubby` 软件包。
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    grubby --default-kernel
+    ```
+
+    ```
+    /boot/vmlinuz-3.10.0-957.el7.x86_64
+    ```
+
+3. 执行 `grubby --update-kernel` 命令修改内核配置：
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    grubby --args="transparent_hugepage=never" --update-kernel /boot/vmlinuz-3.10.0-957.el7.x86_64
```

Comment:
Notice: Certain ktune and/or tuned profiles specify to enable THP when they are applied.